Xamarin.Forms.Platform.Android/VisualElementRenderer.cs

```diff
@@ -238,8 +238,6 @@ protected override void Dispose(bool disposing)
 					}
 				}
 
-				RemoveAllViews();
```

Comment:
this change is my doubt. al lthe rest seems fine

Reply:
Why do we need to call `RemoveAllViews()` if the `ViewGroup` is being disposed? 

If we look at what it does (https://android.googlesource.com/platform/frameworks/base/+/master/core/java/android/view/ViewGroup.java#5536) - it's doing a ton of cleanup work to handle removing focus from the child views, cancelling touches, etc. And if forces a layout pass. But we don't need any of that because the whole ViewGroup is going away. I suspect we're just slowing down `Dispose` (and causing weird bugs) by calling this.